ext/rbs_extension/parser.c

```diff
@@ -2153,17 +2146,17 @@ VALUE parse_attribute_member(parserstate *state, position comment_pos, VALUE ann
   rbs_loc_add_optional_child(loc, rb_intern("ivar_name"), ivar_name_range);
   rbs_loc_add_optional_child(loc, rb_intern("visibility"), visibility_range);
 
-  return rbs_ast_members_attribute(
```

Comment:
Split into 3 types and functions:

1. `rbs_ast_members_attr_reader`
2. `rbs_ast_members_attr_writer`
3. `rbs_ast_members_attr_accessor`